File: Codinsa2015.Client/Codinsa2015.Client/Views/Clank/ClientJava/src/StateAlterationParametersView.java
import java.lang.*;

	public class StateAlterationParametersView
	{

	
		public Vector2 DashTargetDirection;	
		public int DashTargetEntity;	
		public static StateAlterationParametersView Deserialize(BufferedReader input) {
		try {
			StateAlterationParametersView _obj =  new StateAlterationParametersView();
			// DashTargetDirection
			Vector2 _obj_DashTargetDirection = Vector2.deserialize(input);
			_obj.DashTargetDirection = _obj_DashTargetDirection;
			// DashTargetEntity
			int _obj_DashTargetEntity = Integer.Parse(input.readLine());
			_obj.DashTargetEntity = _obj_DashTargetEntity;
			} catch (UnsupportedEncodingExceptio e) { 
			} catch (IOException e) { }
			return _obj;
		}

		public void serialize(OutputStreamWriter output) {
			try {
			// DashTargetDirection
			this.DashTargetDirection.serialize(output);
			// DashTargetEntity
			output.append(((Integer)this.DashTargetEntity).toString() + "\n");
			} catch (UnsupportedEncodingExceptio e) { 
			} catch (IOException e) { }
		}

	}
}

File: Codinsa2015.Client/Codinsa2015.Client/Views/StateAlterationParametersView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Microsoft.Xna.Framework.Graphics;
using Microsoft.Xna.Framework;


namespace Codinsa2015.Views
{

	public class StateAlterationParametersView
	{

static Encoding BOMLESS_UTF8 = new UTF8Encoding(false);
	
		public Vector2 DashTargetDirection;	
		public int DashTargetEntity;	
		public static StateAlterationParametersView Deserialize(System.IO.StreamReader input) {
			StateAlterationParametersView _obj =  new StateAlterationParametersView();
			// DashTargetDirection
			Vector2 _obj_DashTargetDirection = Vector2.Deserialize(input);
			_obj.DashTargetDirection = (Vector2)_obj_DashTargetDirection;
			// DashTargetEntity
			int _obj_DashTargetEntity = Int32.Parse(input.ReadLine());
			_obj.DashTargetEntity = (int)_obj_DashTargetEntity;
			return _obj;
		}

		public void Serialize(System.IO.StreamWriter output) {
			// DashTargetDirection
			this.DashTargetDirection.Serialize(output);
			// DashTargetEntity
			output.WriteLine(((int)this.DashTargetEntity).ToString());
		}

	}
}
